### ISSUE TITLE
CUDA version issue

### ISSUE THREAD

**Issue by yyysst** [co-me-tokens/CoMe#6]
Due to configuration reasons, I am unable to use CUDA 12.8/13.0 and can only use CUDA<=12.6 for inference acceleration verification. However, the experiment found that the inference speed of co_me is slower than mapanything. I see that you have customized the CUDA kernel function. Can only CUDA version 12.8/13.0 be used？

**Comment by yyysst** [comment]
> Hi, [@yyysst](https://github.com/yyysst)
> 
> Could you provide a full list of configuration you are using? Including the CPU and GPU model, CUDA version, and the command you ran (if you are using the provided script in the repo).
> 
> It is hard to guess a reason without these information.
> 
> As for the CUDA compatibility - I have only tested on CUDA 12.8 and 13.0 (All the compute I have access to are in these two versions). Other CUDA versions should work, in principle, but I could not guarantee their functionality.
> 
> Best, Yutian

**My configuration information is as follows:**

PyTorch: 2.6.0+cu124；CUDA used by PyTorch: 12.4；cuDNN version: 90100；GPU: NVIDIA H800

**Eval.mapanything_accelerate_ratio uses synthetic data for testing. I set the height and width (H × W) of each frame to 504 × 378.**

(1) Mapanything:
(come) root@*:/my/project/CoMe# python -m Eval.mapanything_accelerate_ratio ma 32
Loading pretrained dinov2_vitl14 from torch hub
Using cache found in /mnt/cache/torch/hub/facebookresearch_dinov2_main
Loading weights from local directory
warmup ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:04
ma | Sequence length: 32                | Backbone infer time 1341.099ms
(2) Co-Me:
(come) root@*:/my/project/CoMe# python -m Eval.mapanything_accelerate_ratio ours 32
Loading pretrained dinov2_vitl14 from torch hub
Using cache found in /mnt/cache/torch/hub/facebookresearch_dinov2_main
Loading weights from local directory
warmup ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:11
ours | Sequence length: 32              | Backbone infer time 2383.089ms

**Comment by MarkChenYutian** [comment]
@yyysst I see, I think the problem is that the FlexAttention failed to compile on your environment. 

We use flex attention to minimize the overhead of attention bias correction. (FlashAttn does not support attention bias term).

If you only want to verify the speedup one potential option is to change attention backend for both baseline and accelerated version to MemEfficient attention.

**Comment by MarkChenYutian** [comment]
Hi @yyysst , we are planning to release a new version of repository (almost completely rewritten) and we should support CUDA 12.6 w/ GPU compute capability > 8.0. Please stay tuned!

**Comment by yyysst** [comment]
Hi~ We have conducted testing on CUDA13.0 and confirmed that the above flex attention warning is a CUDA version issue. Looking forward to your new version of repository.

**Comment by MarkChenYutian** [comment]
Sounds good, I will close this issue as resolved.
